Разделить, не смешивать

ShabrukovMV opened the pull request:
- Создание модулей для каждой команды
- Создание модуля, загружающего все модули команд (load-commands)
- Расширение функционала, новые команды (--author, --license, --description)

---
:mortar_board: [Разделить, не смешивать](https://up.htmlacademy.ru/nodejs/2/user/822685/tasks/5)

ShabrukovMV committed:
#5 Разделить, не смешивать
- Создание модулей для каждой команды
- Создание модуля, загружающего все модули команд (load-commands)
- Расширение функционала, новые команды (--author, --license, --description)

delagen commented on index.js:
```diff
@@ -1,14 +1,14 @@
 'use strict';
 
```
Какой из файлов входной? Много дублирования

delagen commented on src/load-commands.js:
```diff
@@ -0,0 +1,31 @@
+
+const argument = process.argv[2];
+
+switch (argument) {
```
Можно было бы собрать команды в объект типа {author, description ... } и потом просто проверять наличие или брать команду как commands[input] или еще как, в зависимости как сделаешь ключи объекта
Тут не вижу обработки неизвестной команды

ShabrukovMV committed:
#5 Разделить, не смешивать
- Создание модулей для каждой команды
- Создание модуля, загружающего все модули команд (load-commands)
- Расширение функционала, новые команды (--author, --license, --description)
- Исправления по замечаниям наставника
